### PR TITLE
fix: ci should be run on PR via types

### PR DESCRIPTION
**Pull request by aarnphm**
Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by aarnphm** [comment]
should address #2640

**Comment by sauyon** [comment]
you need to add synchronized here, but I'm not sure it's broken?

**Review by ssheng — DISMISSED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
   pull_request:
+    types: [opened, reopened]
```
Comment (aarnphm) [inline .github/workflows/ci.yml:8]:
```suggestion
    types: [opened, reopened, synchronized]
```

**Comment by sauyon** [comment]
Closing, I think this wasn't a problem.

**Pull request closed without merging**